package.json

```diff
@@ -9,7 +9,7 @@
     "tsc": "cross-env NODE_ENV=development ./node_modules/.bin/tsc",
     "checks": "concurrently 'yarn lint' 'yarn test' 'yarn tsc'",
     "postinstall": "node -r @babel/register ./config/scripts/CheckNativeDep.js && electron-builder install-app-deps && yarn cross-env NODE_ENV=development webpack --config ./config/webpack/webpack.config.renderer.dev.dll.babel.js && yarn-deduplicate yarn.lock",
-    "start": "NODE_ENV=production node -r @babel/register ./config/scripts/CheckPortInUse.js && cross-env yarn start:renderer",
+    "start": "NODE_ENV=production env-cmd -f env/electron-publish.env node -r @babel/register ./config/scripts/CheckPortInUse.js && cross-env yarn start:renderer",
```

Comment:
so is the file `.env` or `env/electron-publish.env`. Is `env/electron-publish.env` create by webpack or smth? I would add some reference to the `dev` command below too.

Reply:
```
➜  app git:(visual_adjustments) ✗ yarn start
yarn run v1.22.10
$ NODE_ENV=production env-cmd -f env/electron-publish.env node -r @babel/register ./config/scripts/CheckPortInUse.js && cross-env yarn start:renderer
Error: Failed to find .env file at path: env/electron-publish.env
    at getEnvFile (/Users/dariah/Documents/Dev/tokel/app/node_modules/env-cmd/dist/get-env-vars.js:40:19)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```